src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -96,18 +126,20 @@
                            (if (= type "array")
                              (str "Comma-separated list of " (:type items) "s")
                              (clojure.string/capitalize type))])])
-                     (if enum
-                       (style/create-identity-select {:ref pk-str
-                                                      :defaultValue (get existing property-key ENUM_EMPTY_CHOICE)}
-                                                     (cons ENUM_EMPTY_CHOICE enum))
-                       [input/TextField {:ref pk-str
-                                         :style {:width "100%"}
-                                         :placeholder inputHint
-                                         :defaultValue (render-value (get existing property-key default))
-                                         :predicates [(when required?
-                                                        (input/nonempty pk-str))
-                                                      (when (= type "integer")
-                                                        (input/integer pk-str :min minimum))]}])])))
+                     (cond
+                       enum (style/create-identity-select {:ref pk-str
+                                                             :defaultValue (get existing property-key ENUM_EMPTY_CHOICE)}
+                                                            (cons ENUM_EMPTY_CHOICE enum))
+                       (= typeahead "ontology") [:select {:ref "duos-typeahead"
+                                                          :style {:width "100%"}}]
```

Comment:
what happens if there are two attributes with typeahead? They'd both be looking for the ref `duos-typeahead`.